docs/docs/contributor/server/others/best-practices.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Best Practices
+sidebar_position: 3
+sidebar_custom_props:
+  icon: TbChecklist
+---
+
+This document outlines the best practices you should follow when working on the backend.
+
+## Follow a modular approach
+
+We follow a modular approach for our backend, which is a fundamental principle when working with NestJS. Make sure you break down your code into reusable modules to maintain a clean and organized codebase. 
+Each module should encapsulate a particular feature or functionality and have a well-defined scope. This modular approach enables clear separation of concerns and removes unnecessary complexities.  
+
+Most of our modules are directly related to a model in the database. For instance, we have a module for users, which is directly related to the user model in the database. 
+
+## Expose services to be used in modules
+
+We encourage creating services that have a clear and single responsibility, which enhances code readability and maintainability. Services should be named descriptively and consistently. 
+
+We also encourage exposing services to be used in other modules. Exposing services to other modules is made possible through NestJS's powerful dependency injection system, and promotes loose coupling between components. 
+
+
+## Custom ESLint rules
+
+In addition to NestJS recommended rules, we use custom ESLint rules to organize the code and make it easy to maintain. If you import a library that's not used in the file, it's going to be removed automatically. We also have rules to reorder `import` statements. We have the NestJS imports first, followed by external imports and then internal imports.    
```

Comment:
Nit: I feel like this part is not necessary here since we don't have very unique rules, also the rule you are describing is auto-fixable and pretty standard I think. If we really want to talk about rules, we should probably talk about some very specific ones that need attention (and can't be fixed with a ctrl+s) and explanations. However I don't have any in mind right now which is why I would drop that section.
What do you think?